src/react-components/src/__tests__/setupEnzyme.ts

```diff
@@ -1,5 +1,34 @@
-
 import { configure } from "enzyme";
 const Adapter = require("enzyme-adapter-react-16");
+const { JSDOM } = require("jsdom");
+
+const jsdom = new JSDOM("<!doctype html><html><body></body></html>");
+const { window } = jsdom;
+
+function copyProps(src, target) {
```

Comment:
For deep copy?

Reply:
Every time I think I need JSDOM, it turns out I don't.  I'm gonna take all of it out for now, enzyme should be able to do all that I need ...

Reply:
I don't think it actually needs this, just the re-assignment of Windows and stuff